pkg/config/config.go

```diff
@@ -327,8 +328,21 @@ func (kubicCfg KubicInitConfiguration) IsSeeder() bool {
 // GetBindIP gets a valid IP address where we can bind
 func (kubicCfg KubicInitConfiguration) GetBindIP() (net.IP, error) {
 	if len(kubicCfg.Network.Bind.Interface) > 0 {
-		// TODO: not implemented yet: get the IP address for that interface
-		return net.IP{}, nil
+		link, err := netlink.LinkByName(kubicCfg.Network.Bind.Interface)
+		if err != nil {
+			return nil, err
+		}
+		addrs, err := netlink.AddrList(link, netlink.FAMILY_V4)
+		if err != nil {
+			return nil, err
+		}
+		for _, addr := range addrs {
+			// return 1st IPv4 address
+			if len(addr.IP) >= 4 {
+				return addr.IP, nil
+			}
+		}
+		return nil, fmt.Errorf("No address found in %s", kubicCfg.Network.Bind.Interface)
```

Comment:
I think maybe we can avoid the use of `netlink` and stick to the standar library with something like this:

```suggestion
		ifaces, err := net.Interfaces()
		if err != nil {
		    return nil, err
		}

		for _, i := range ifaces {
		    if strings.ToLower(i.Name) != strings.ToLower(kubicCfg.Network.Bind.Interface) {
		        continue
		    }
    
		    addrs, err := i.Addrs()
		    if err != nil {
		        return nil, err
		    }

		    for _, addr := range addrs {
		        var ip net.IP
		        // just return the first IP (maybe we could do some smarter heuristics...)
		        switch v := addr.(type) {
		        case *net.IPNet:
		                return v.IP, nil
		        case *net.IPAddr:
		                return v.IP, nil
		        }
		    }
		}

		return nil, fmt.Errorf("No address found in %s", kubicCfg.Network.Bind.Interface)

```

Reply:
Cool, much better!